packages/ckeditor5-build-helpjuice-v2/src/hjeditor.ts

```diff
@@ -349,6 +349,18 @@ HelpjuiceEditor.defaultConfig = {
 						`<video controls="" style="max-width: 100%" src="${getUrl}">Your browser does not support HTML5 video.</video>`
 					);
 				}
+			},
+			{
+				name: 'loomProvider',
+				url: new RegExp(`^https:\\/\\/www\\.loom\\.com\\/share\\/(\\w+)`),
+				html: match => {
+					const getUrl = match.input;
+					const videoId = getUrl?.split('/').pop();
+
+					return (
+						`<div style="position: relative; padding-bottom: 100%; height: 0; padding-bottom: 56.2493%;"><iframe src="https://www.loom.com/embed/${videoId}" style="position: absolute; width: 100%; height: 100%; top: 0; left: 0;" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen=""></iframe></div>`
```

Comment:
Do we need the div?

Reply:
@SpaYco This div use to style the loom video in the editor's view, so yes we need it.

I checked `mediaembedediting.ts` file and they use the same for youtube videos

![Screenshot from 2023-08-16 23-18-17](https://github.com/Helpjuice/ckeditor5/assets/45231276/7f460268-6fa8-4bc6-b958-f540e4a784b5)

Reply:
Not sure why they are using two `padding-bottom` here, so I guess I can remove the first one `padding-bottom: 100%;"`